### PR TITLE
Deployment Translated. Please merge.

### PR DESCRIPTION
**Pull request by yoheisugiyama**

### CODE REVIEW

File: ja/deployment.rst [hunk 1]
```diff
@@ -1,22 +1,192 @@
...
+アプリケーションでドキュメントルートを正しく指定することはコードをセキュアに、 またアプリケーションを安全に保つために重要なステップの内の一つです。CakePHP のアプリケーションは、 アプリケーションの ``webroot`` にドキュメントルートを指定する必要があります。 これによってアプリケーション、設定のファイルが URL を通してアクセスすることができなくなります。 ドキュメントルートの指定の仕方はウェブサーバーごとに異なります。 WEBサーバー特有の情報については :ref:`url-rewriting` ドキュメントを見てください。
+
+どの場合においても ``webroot/`` をバーチャルホスト（バーチャルドメイン）のドキュメントルートに 設定すべきでしょう。これは webroot ディレクトリの外側のファイルを実行される可能性を取り除きます。
 
 .. _symlink-assets:
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
+>>>>>>> 487a61651c169fb42bc5d592f5d7fff60b2e058a
```
Comment (HavokInspiration) [inline ja/deployment.rst]: Those are traces of rebase that should be removed.

File: ja/deployment.rst [hunk 2]
```diff
@@ -1,22 +1,192 @@
...
+
+もしあなたのファイルシステムはシンボリックリンクを作成することを許可しないようであれば、ディレクトリーはシンボリックリンクが貼られる代わりにコピーされます。また、以下を使用しながら、特定のディレクトリーをコピーすることができます。
+
+::
+
+    bin/cake plugin assets copy
+=======
 Improve Your Application's Performance
 ======================================
+>>>>>>> 80d29b995076f448170eff6cd238852092e8661d
```
Comment (HavokInspiration) [inline ja/deployment.rst]: Those are traces of rebase that should be removed.

File: ja/deployment.rst [hunk 3]
```diff
@@ -1,22 +1,192 @@
...
 
 .. meta::
-    :title lang=ja: デプロイ
-    :keywords lang=ja: stack traces,application extensions,set document,installation documentation,development features,generic error,document root,func,debug,caches,error messages,configuration files,webroot,deployment,cakephp,applications
+    :title lang=en: Deployment
+<<<<<<< HEAD
+    :keywords lang=en: stack traces,application extensions,set document,installation documentation,development features,generic error,document root,func,debug,caches,error messages,configuration files,webroot,deployment,cakephp,applications
+=======
+    :keywords lang=en: stack traces,application extensions,set document,installation documentation,development features,generic error,document root,func,debug,caches,error messages,configuration files,webroot,deployment,cakephp,applications
+>>>>>>> 487a61651c169fb42bc5d592f5d7fff60b2e058a
```
Comment (HavokInspiration) [inline ja/deployment.rst]: Those are traces of rebase that should be removed.

**Comment by antograssiot** [comment]
merging, I'll fix the last warning post-merge.
Thank you @yoheisugiyama

**Pull request merged**